docs/android/get-started/installation/index.md

```diff
@@ -54,6 +54,7 @@ This article explains how to setup a physical Android device and
 connect it to a development computer so that the device may be used to
 run and debug Xamarin.Android applications.
 
-## [Preview with Microsoft's OpenJDK Distribution](~/android/get-started/installation/openjdk.md)
```

Comment:
@mgmclemore we can't use "OpenJDK" by itself without violating the trademark. It has to be a Microsoft distribution of OpenJDK

Reply:
Can we use **Microsoft OpenJDK Preview**?  We need to try to keep the TOC titles reasonably short.